### PR TITLE
cockpit 226, cockpit-podman 22

### PR DESCRIPTION
**Pull request by mvollmer**
https://mvollmer.github.io/cockpit-project.github.io/blog/cockpit-226.html

### CODE REVIEW

File: _posts/2020-08-19-cockpit-226-cockpit-podman-22.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 226 and Cockpit Podman version 22.
+
+### Storage: Better support for "noauto" LUKS devices
+
+Cockpit now behaves more sensible with encrypted filesystems that have
+"noauto" in their LUKS options. Such a filesystem will now also always
+have "noauto" in their mounting options even when mounted.
```
Comment (garrett) [inline _posts/2020-08-19-cockpit-226-cockpit-podman-22.md:18]:
```suggestion
Cockpit  Encrypted filesystems with "noauto" in LUKS options now .... ??!??!?
```

This needs rewriting, but it doesn't say what Cockpit did before or what it does now.

Comment (garrett) [reply _posts/2020-08-19-cockpit-226-cockpit-podman-22.md:18]:
```suggestion
Cockpit now better respects the "noauto" option for encrypted filesystems.

Previously, Cockpit would force a "noauto" encrypted filesystem to mount at next boot when it was simply mounted within Cockpit. Now, a filesystem with the "noauto" settings will remain unmounted.
```

File: _posts/2020-08-19-cockpit-226-cockpit-podman-22.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+### Podman: Support for pod group deletion
+
+The cockpit-podman plugin gained the ability to delete pod groups.
+
+![Pod group deletion](/images/podman-pod-deletion.png){:.screenshot}
```
Comment (garrett) [inline _posts/2020-08-19-cockpit-226-cockpit-podman-22.md:24]:
```suggestion
![Pod group deletion](/images/podman-pod-deletion.png)
```

We don't necessarily need the screenshot class on a screenshot. It's intended to add a border around partial screenshots. This screenshot already has a border inside the image itself.

(Perhaps I should rename the class.)

File: _posts/2020-08-19-cockpit-226-cockpit-podman-22.md (hunk 1 above)
Comment (garrett) [reply _posts/2020-08-19-cockpit-226-cockpit-podman-22.md:18]: Another try:

```suggestion
Cockpit now better respects the LUKS "noauto" option for encrypted filesystems.

Previously, Cockpit would force a encrypted filesystem to mount at next boot when it was simply mounted within Cockpit. Now, encrypted filesystems set with "noauto" will remain unmounted during boot.

Please [refer to issue 14467](https://github.com/cockpit-project/cockpit/issues/14467) for more details.
```

File: _posts/2020-08-19-cockpit-226-cockpit-podman-22.md (hunk 2 above)
Comment (mvollmer) [reply _posts/2020-08-19-cockpit-226-cockpit-podman-22.md]: Ah, I see! I just copied this from the previous release notes without understanding what it does.

**Comment by allisonkarlitskaya** [comment]
Looks fine to me.  Seems like the shell/fd cleanup stuff didn't make it after all :rofl:

**Comment by mvollmer** [comment]
Thanks everyone!

**Pull request merged**